orchagent/vnetorch.cpp

```diff
@@ -509,9 +509,14 @@ bool VNetOrch::addOperation(const Request& request)
             else
             {
                 SWSS_LOG_NOTICE("VNET '%s' already exists ", vnet_name.c_str());
+                if (overlay_dmac != it->second->getOverlayDMac())
```

Comment:
please check overlay_dmac !empty. If the second config comes as "empty" mac, then we should not do anything.

Reply:
Sure. I'll fix it.

Reply:
done